.htaccess

```diff
@@ -156,14 +156,67 @@
+	# RewriteRule ^framework/en/[\d.]*/contributing/?(.*) http://doc.silverstripe.org/framework/en/trunk/contributing/$1 [R=301,L]
+
+
+	#### Issue 50 : Top 50 404 Errors ####
+	RewriteRule ^changelog/?$ /en/changelogs [R=301,L]
+	RewriteRule ^installation-into-subversion/?$ /en/getting_started/composer [R=301,L]
+	# check target with Cam
+	RewriteRule ^modules:rssaggregator/?$ http://addons.silverstripe.org/add-ons/silverstripe/versionfeed [R=301,L]
+	RewriteRule ^memberauthenticator/?$ /en/developer_guides/security/authentication [R=301,L]
+	# check target with Cam
+	RewriteRule ^modules:mssql/?$ http://addons.silverstripe.org/add-ons/silverstripe/mssql [R=301,L]
+	RewriteRule ^authenticator/?$ /en/developer_guides/security/authentication [R=301,L]
+	RewriteRule ^recipes:combining_files/?$ /en/developer_guides/templates/requirements [R=301,L]
+	RewriteRule ^datefield/?$ /en/developer_guides/forms/field_types/datefield [R=301,L]
+	RewriteRule ^member/?$ /en/developer_guides/security/member [R=301,L]
+	RewriteRule ^en/topics/datamodel/?$ /en/developer_guides/model/data_model_and_orm [R=301,L]
+	RewriteRule ^treedropdownfield/?$ /en/developer_guides/forms/field_types/common_subclasses [R=301,L]
+	RewriteRule ^(.*)/tutorials/1-building-a-basic-site/?$ /$1/tutorials/building-a-basic-site [R=301,L]
+	RewriteRule ^en/topics/security/?$ /en/developer_guides/security [R=301,L]
+	RewriteRule ^en/topics/environment-management/?$ /en/getting_started/environment_management [R=301,L]
+	RewriteRule ^en/topics/testing/?$ /en/developer_guides/testing [R=301,L]
+	RewriteRule ^(.*)/tutorials/2-extending-a-basic-site/?$ /$1/tutorials/extending-a-basic-site [R=301,L]
+	RewriteRule ^en/topics/configuration/?$ /en/developer_guides/configuration [R=301,L]
+	RewriteRule ^download/?$ http://www.silverstripe.org/software/download [R=301,L]
+	RewriteRule ^en/reference/permission/?$ /en/developer_guides/security/permissions [R=301,L]
+	RewriteRule ^en/topics/controller/?$ /en/developer_guides/controllers [R=301,L]
+	RewriteRule ^typography/?$ /en/developer_guides/customising_the_admin_interface/typography [R=301,L]
+	RewriteRule ^en/howto/?$ /en/tutorials [R=301,L]
+	RewriteRule ^en/topics/rich-text-editing/?$ /en/developer_guides/forms/field_types/htmleditorfield [R=301,L]
+	RewriteRule ^en/reference/modeladmin/?$ /en/developer_guides/customising_the_admin_interface/modeladmin [R=301,L]
+	RewriteRule ^(.*)/tutorials/5-dataobject-relationship-management/?$ /$1/tutorials/dataobject-relationship-management [R=301,L]
+	RewriteRule ^(.*)/tutorials/3-forms/?$ /$1/tutorials/forms [R=301,L]
+	RewriteRule ^en/2.4/topics/reference/requirements/?$ http://docs.silverstripe.org/en/2.4/reference/requirements [R=301,L]
+	# check target with Cam
+	RewriteRule ^modules:auth_ext_drivers/?$ https://github.com/mattclegg/silverstripe-doc-restructuring/blob/master/output/modules/auth_ext_drivers.md [R=301,L]
```

Comment:
Hmmm perhaps in these "modules:*" cases maybe we have a standard to link to them on addons if they exist otherwise just send the visitor to the home of the addons site?